anker/src/instruction.rs

```diff
@@ -381,3 +399,95 @@ pub fn change_token_swap_pool(
         data: data.to_vec(),
     }
 }
+
+accounts_struct! {
+    SendRewardsAccountsMeta, SendRewardsAccountsInfo {
+        pub anker {
+            is_signer: false,
+            is_writable: false,
+        },
+        pub solido {
+            is_signer: false,
+            is_writable: false,
+        },
+        pub reserve_authority {
+            is_signer: false,
+            is_writable: false,
+        },
+        // Accounts for Wormhole swap.
+        pub wormhole_token_bridge_program_id {
+            is_signer: false,
+            is_writable: false,
+        },
+        pub wormhole_core_bridge_program_id {
+            is_signer: false,
+            is_writable: false,
+        },
+        pub payer {
+            is_signer: true,
+            is_writable: true,
+        },
+        pub config_key {
+            is_signer: false,
+            is_writable: false,
+        },
+        // From should be the UST token address.
```

Comment:
This comment makes little sense now after renaming the account. Maybe say something like

> The `ust_reserve_account` is the account that we send the funds from for the wormhole transfer.